.storybook/theme-selector/index.js

```diff
@@ -53,8 +53,14 @@ export const withThemeSelector = makeDecorator({
     channel.emit(PARAMS_EVENT, parameters);
     return (
       <ThemeProvider theme={theme}>
-        {!parameters.disable && <CarbonGlobalTokensProvider />}
-        {getStory(context)}
+        {parameters.disable ? (
+          getStory(context)
+        ) : (
+          <CarbonScopedTokensProvider>
+            {" "}
```

Comment:
Looks like something's gone weird with the formatting, we can remove the empty strings here